OnStreetParking/examples/example-normalized.json.csv

```diff
@@ -1,2 +1,2 @@
-"id_", "type_", "category__value__0_", "category__value__1_", "category__value__2_", "permitActiveHours__value__blueZonePermit_", "requiredPermit__value__0_", "requiredPermit__value__1_", "allowedVehicleType__value_", "chargeType__value__0_", "refParkingGroup__type_", "refParkingGroup__value__0_", "refParkingGroup__value__1_", "totalSpotNumber__value_", "location__type_", "location__value__type_", "location__value__coordinates__0__0__0_", "location__value__coordinates__0__0__1_", "location__value__coordinates__0__1__0_", "location__value__coordinates__0__1__1_", "location__value__coordinates__0__2__0_", "location__value__coordinates__0__2__1_", "location__value__coordinates__0__3__0_", "location__value__coordinates__0__3__1_", "location__value__coordinates__0__4__0_", "location__value__coordinates__0__4__1_", "areaServed__value_", "maximumAllowedStay__value_", "dateModified__type_", "dateModified__value_", "extraSpotNumber__value_", "availableSpotNumber__value_", "availableSpotNumber__metadata__timestamp__value_", "availableSpotNumber__metadata__timestamp__type_"
-"santander:daoiz_velarde_1_5", "OnStreetParking", "blueZone", "shortTerm", "forDisabled", "Mo, Tu, We, Th, Fr, Sa 09:00-20:00", "blueZonePermit", "disabledPermit", "car", "temporaryFee", "Relationship", "daoiz-velarde-1-5-main", "daoiz-velarde-1-5-disabled", "6", "geo:json", "Polygon", "-3.80356167695194", "43.46296641666926", "-3.803161973253841", "43.46301091092682", "-3.803147082548618", "43.462879859445884", "-3.803536474744068", "43.462838666196674", "-3.80356167695194", "43.46296641666926", "Zona Centro", "PT2H", "DateTime", "2016-06-02T09:25:55.00Z", "2", "3", "2018-09-12T12:00:00", "DateTime"
\ No newline at end of file
+"id_", "type_", "category__value__0_", "category__value__1_", "category__value__2_", "permitActiveHours__value__blueZonePermit_", "requiredPermit__value__0_", "requiredPermit__value__1_", "allowedVehicleType__value_", "chargeType__value__0_", "refParkingGroup__type_", "refParkingGroup__value__0_", "refParkingGroup__value__1_", "totalSpotNumber__value_", "location__type_", "location__value__type_", "location__value__coordinates__0__0__0_", "location__value__coordinates__0__0__1_", "location__value__coordinates__0__1__0_", "location__value__coordinates__0__1__1_", "location__value__coordinates__0__2__0_", "location__value__coordinates__0__2__1_", "location__value__coordinates__0__3__0_", "location__value__coordinates__0__3__1_", "location__value__coordinates__0__4__0_", "location__value__coordinates__0__4__1_", "areaServed__value_", "maximumAllowedStay__value_", "dateModified__type_", "dateModified__value_", "extraSpotNumber__value_", "availableSpotNumber__value_", "availableSpotNumber__metadata__timestamp__value_", "availableSpotNumber__metadata__timestamp__type_", "occupiedSpotNumber__value_", "occupiedSpotNumber__metadata__timestamp__value_", "occupiedSpotNumber__metadata__timestamp__type_"
```

Comment:
this file is generated automatically so it is not necessary (either convenient to be in the PR)